issuer.go

```diff
@@ -0,0 +1,111 @@
+package issuer
+
+import (
+	"bytes"
+	"compress/flate"
+	"crypto/ecdsa"
+	"encoding/json"
+	"github.com/skip2/go-qrcode"
+	"gopkg.in/square/go-jose.v2"
+	"strconv"
+	"time"
+)
+
+const (
+	MAX_SINGLE_JWS_SIZE = 1195
+	MAX_CHUNK_SIZE      = 1191
+)
+
+type SmartHealthCard struct {
+	IssuerUrl            string                 `json:"iss"`
+	IssuanceDate         int                    `json:"nbf"`
+	VerifiableCredential map[string]interface{} `json:"vc"`
+}
+
+type IssueCardInput struct {
+	IssuerUrl            string
+	PrivateKey           *ecdsa.PrivateKey
+	KeyId                string
+	VerifiableCredential map[string]interface{}
+}
+
+func IssueCard(input IssueCardInput) (string, error) {
```

Comment:
Do we need this function? Or can we just have package users instantiate a `SmartHealthCard`? What is the difference between the output of `Sign` and `CompactSerialize`? Could we leave it up to the user to call `CompactSerialize`?

The only other thing that I can think of is that it might make sense to use a `NewCard` function that returns a `SmartHealthCard` with private fields. This allows us to do some validation on the inputs... but I'm not sure if it's necessary.

Reply:
`Sign` returns `(*jose.JSONWebSignature, error)`, while `CompactSerialize` returns `(string, error)`. 

I suppose we could just let the user create an instance of a `SmartHealthCard` struct. We'd then just need to document that they need to call the card's `Sign` method to produce the jws value that's ultimately used in the real world context. We could move the call to `CompactSerialize` inside the `Sign` method so they don't have to call that themselves too.

I guess it comes down to how much work we want this package to do for the user. I think by encapsulating these steps in the `IssueCard` method, we save other services from having to reimplement the same steps. But it might be better to give users more flexibility. Let's discuss more tomorrow!